Cargo.toml

```
@@ -22,7 +22,7 @@ http = "0.2.1"
 sha2 = "0.10.1"
 base64 = "0.12.0"
 reqwest = {version = "0.11", features = ["blocking"], optional = true}
-rouille = {version = "3.0.0", optional = true}
+rouille = { git = "ssh://git@github.com/PassFort/rouille.git", branch = "pf-master" , optional = true}
```

I don't think github actions likes our internal/forked crates. Best to just:

```suggestion
rouille = { version = "4", branch = "pf-master" , optional = true }
```

---

(psst the fork you're trying to download is private - you need to make the `github.com/PassFort/rouille.git` repo public or figure out a way to setup auth in this github action)